### ISSUE TITLE
logging not appearing on Node.js for optimizations other than none 

### ISSUE THREAD

**Issue by eval** [harto/huon#6]
Code demonstrating this: https://github.com/eval/huon/tree/logging-disabled-for-simple#logging-issue-when-targetting-nodejs

tl;dr:
- logging is shown for optimization none (-t node)
- logging is not shown for other optimizations (-t node)
- there appears no difference when not targetting node

## origin

I found that `goog.debug.Console.console_` is null for optimization != none whereas it equals `js/console` for optimization == none.  
relevant line: https://github.com/google/closure-library/blob/master/closure/goog/debug/console.js#L156

_speculation_ it appears to be an load ordering issue with ClojureScript.

## possible workaround

As part of enabling logging, we should setConsole to `js/console` (see code in issue).

**Comment by eval** [comment]
Cljs-issue: https://dev.clojure.org/jira/browse/CLJS-2930

**Comment by harto** [comment]
Thanks for the detailed report and steps to reproduce!

I wonder if we can rig up a regression test somehow. Maybe we can change the test runner to work with different targets & optimization levels.

**Comment by harto** [comment]
Released as 0.5.1. Thanks again!
